mpegdash/nodes.py

```diff
@@ -335,6 +335,45 @@ def write(self, xmlnode):
         write_attr_value(xmlnode, 'ns2:default_KID', self.key_id)
 
 
+class PSSH(XMLNode):
+    def __init__(self):
+        self.pssh = None
+
+    def parse(self, xmlnode):
+        self.pssh = parse_node_value(xmlnode, str)
+
+    def write(self, xmlnode):
+        write_node_value(xmlnode, self.pssh)
+
+
+class ContentProtectionComponent(XMLNode):
+    def __init__(self):
+        self.scheme_id_uri = ""                            # xs:anyURI (required)
+        self.value = None                                  # xs:string
+        self.id = None                                     # xs:string
+        self.pssh = None                                   # PSSH
+        self.default_key_id = None                         # xs:string
+        self.ns2_key_id = None                             # xs:string
+        self.cenc_key_id = None                            # xs:string
```

Comment:
I have a question. `ns2_key_id` and `cenc_key_id` are only used in <ContentProtection .. /> tags?

If so, I think we can move `Descriptor::ns2_key_id` to ContentProtection class to make `Descriptor` slim. Also, #52 could be covered this, right?

Reply:
Yes